strapi/strapi
# refactor(core/strapi): move components to registry

### What does it do?

Move the loading and fetching of components through a registry

### Why is it needed?

For future feature that will allow to load components from plugins

### How to test it?

Follow the default instructions how to run Strapi locally, and check components in content builder

### Related issue(s)/PR(s)

PR (Draft): https://github.com/strapi/strapi/pull/16132

PR (Closed): https://github.com/strapi/strapi/pull/12965
PR (Closed): https://github.com/strapi/strapi/pull/9104
Issue (Closed): https://github.com/strapi/strapi/issues/7640
Feature request: [feedback.strapi.io/developer-experience/p/components-inside-of-plugins](https://feedback.strapi.io/developer-experience/p/components-inside-of-plugins)
